src/TLuaInterpreter.cpp

```diff
@@ -309,6 +309,11 @@ int TLuaInterpreter::raiseEvent( lua_State * L )
             pE.mArgumentList.append( QString(lua_tostring( L, i )) );
             pE.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
         }
+        else if( lua_isboolean( L, i ) )
+        {
+            pE.mArgumentList.append( QString::number(lua_toboolean( L, i )) );
```

Comment:
As I see it, when examining a TEvent in the Debugger you would have to cross-check the `mArgumentTypeList` at a particular argument index to check whether the string "0" in the `mArgumentList` at that index is a number 0 or a boolean false.  It is just that 0 or 1 are *numbers* that are quite likely to be used whereas my alternative of using "false" or "true" as the boolean values are much less likely to be mixed up with actual strings with those same values - because they are *less likely to actually arise in practice* as **strings** a user will want to use in their scripts, IMO.